pkg/prototype/rice-box.go

```diff
@@ -1,18 +1,3 @@
-// Copyright 2018 The ksonnet authors
```

Comment:
Keeping the copyright notice on generated files has been somewhat a pain point.

Reply:
It looks like it's at least addressable with _mockery_ - something to look at later:

```
  -note string
    	comment to insert into prologue of each generated file
```